kedro/extras/extensions/ipython.py

```diff
@@ -44,17 +47,26 @@ def reload_kedro(path, env: str = None, extra_params: Dict[str, Any] = None):
     from kedro.framework.session.session import _activate_session
     from kedro.framework.startup import bootstrap_project
 
-    path = path or project_path
-    metadata = bootstrap_project(path)
+    # If a path is provided, set it as default for subsequent calls
+    global default_project_path
+    if path:
+        default_project_path = Path(path).expanduser().resolve()
+        logging.info("Updated path to Kedro project: %s", str(default_project_path))
+    else:
+        logging.info(
+            "No path argument was provided. Using: %s", str(default_project_path)
```

Comment:
Very minor: I suspect these `str` calls could be removed as the logging `%s` would coerce to string anyway (done in several places throughout file).